.circleci/config.yml

```diff
@@ -51,6 +77,7 @@ workflows:
   version: 2
   build:
     jobs:
+      - linkchecker
```

Comment:
So this will run on every pr? 

What happens when a link breaks that's totally unrelated to this PR? Can we have this job as a 'warn' job not a 'fail' job?

Reply:
This will run on every PR, yes. 

Warnings have a tendency to be ignored over time - if unrelated breaking links blocking PRs become a problem, we can revisit but let's not meet trouble halfway. 

This issue has not arisen on www.ubuntu.com and this check has been in place for a year or more if memory serves me correctly.

Reply:
@hatched 	yes it will run on every pr and saved us on modifications that seemed small but impacted other places... The PR might be longer to be "validated" from the CI but we should use this.